libc/src/__support/macros/config.h

```diff
@@ -28,6 +28,7 @@
 #define LIBC_HAS_FEATURE(f) 0
 #endif
 
+#ifndef __GNUC__
```

Comment:
doesn't clang define `__GNUC__` for compatibility? Would it be better to instead use `#ifdef __clang__` instead?

Reply:
Done.